### ISSUE TITLE
File search should allow (linenum): suffix

### ISSUE THREAD

**Issue by randomascii** [chromium/vs-chromium#21]
Visual studio error/warnings contain the line number in parentheses at the end of the file path. This allows double-clicking to go to the file/line number. However if the error/warning is from a different machine or is partial (see https://code.google.com/p/chromium/issues/detail?id=440500#c189 and #c185 for examples) then navigating to the correct place in the code requires copy/paste of path into VsChromium, F8 to go there, then back to the source text to copy the line number, then back to VS, Ctrl+G, paste, enter to go to the line number.

If VsChromium understood the (linenum) syntax then it could ignore it during searches and go to that line number on F8/double-click.

It's a little bit inelegant, but the value is great enough to make it worthwhile overlooking this.

The colon that often follows the (linenum) should also be ignored if it is present.

**Comment by rpaquay** [comment]
Sounds reasonable, I ran into this myself before.

Adding example in bug description here to avoid having to go to URL.

Example:
```
chromium\src\third_party\webrtc\base\criticalsection.cc(91): warning C4312: ...
```

**Comment by randomascii** [comment]
One significant use-case would be looking through /analyze warnings produced by the build machine. Right now the output has to be search/replaced to match my local paths. This would make it easy to use it unchanged.

**Comment by rpaquay** [comment]
I see. So, this would also requires some sort of heuristics to partially skip the path prefix that comes from the copy/pasted path.

For example, if pasting `chromium\src\third_party\webrtc\base\criticalsection.cc(91)`, we should recognize that `third_party\webrtc` is a top level directory of the local enlistment and remove the "chromium\src" prefix from the path when searching, right?

**Comment by randomascii** [comment]
Skipping the prefix is a nice-to-have, but I don't think it's required. I can easily select a subset of the string in order to skip the prefix - it's going back for the number which is tedious.

So, skipping the prefix (showing files that match except for the prefix) would be nice, but if it's ugly or inefficient then I don't think it's a big deal.

**Comment by rpaquay** [comment]
Implemented in new (beta) release (https://github.com/chromium/vs-chromium/releases/tag/v0.9.17). Let me know if it does not work for your use case.

**Comment by randomascii** [comment]
It works perfectly. Thanks! I wish I'd thought of suggesting it earlier.

**Comment by rpaquay** [comment]
Ok, I published 0.9.17 as the new stable version.
